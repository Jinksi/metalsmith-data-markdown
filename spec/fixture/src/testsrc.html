<section>
<section id="paper-esecurity-cover" data-markdown>
### WLAN Security Revisited (paper)
#### Rogue AP, Deauthentication Attack and Portal via PwnSTAR in 2 Minutes
published on June 2012
<small>FH JOANNEUM information management master</small>
</section>
<section id="paper-esecurity-video" data-markdown>
<iframe width="640" height="480" src="//www.youtube.com/embed/vy4leX_twTw" frameborder="0"></iframe>
*"Creating a Rogue Access Point with redirecting web portal and hijacking open wifi users via de-authentication attack."*
</section>
</section>