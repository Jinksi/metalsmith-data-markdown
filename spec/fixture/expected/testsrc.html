<section>
<section id="paper-esecurity-cover"><h3 id="wlan-security-revisited-paper-">WLAN Security Revisited (paper)</h3>
<h4 id="rogue-ap-deauthentication-attack-and-portal-via-pwnstar-in-2-minutes">Rogue AP, Deauthentication Attack and Portal via PwnSTAR in 2 Minutes</h4>
<p>published on June 2012<br><small>FH JOANNEUM information management master</small></p>
</section>
<section id="paper-esecurity-video"><p><iframe width="640" height="480" src="//www.youtube.com/embed/vy4leX_twTw" frameborder="0"></iframe><br><em>&#x201C;Creating a Rogue Access Point with redirecting web portal and hijacking open wifi users via de-authentication attack.&#x201D;</em></p>
</section>
</section>